packs/equipment/palanquin-of-night.json

```diff
@@ -6,7 +6,7 @@
+            "value": "<p>This small square of dark blue fabric appears to be painted with a depiction of the night sky, shifting each day to match the sky above Geb the previous night.</p>\n<p><strong>Activate</strong> <span class=\"action-glyph\">T</span> Interact</p>\n<hr />\n<p><strong>Effect</strong> The fabric unfolds into a dark palanquin large enough to comfortably hold two Medium creatures. Four spectral bearers appear at the corners of the palanquin, capable of carrying it effortlessly over most types of terrain as long as the terrain is relatively flat and devoid of hazards. Spectral bearers have the statistics of an @UUID[Compendium.pf2e.pathfinder-bestiary.Actor.Unseen Servant], except they're strong enough to carry the palanquin and its occupants. They move generally along a basic set of directions you indicate upon activation, which can include turns onto various streets, but they don't take other creatures' feelings into account and can be inconsiderate to passersby in an inhabited area as they relentlessly move you forward. The bearers move slowly but steadily along their simple directions. While the palanquin is perfectly suitable for overland travel, the bearers' imprecise movements and the requirement to reactivate the palanquin to change destinations make it unsuited for a combat encounter or other situation where seconds and precise movements count.</p>\n<p>The palanquin's interior ceiling is painted with the same night sky pattern as the cloth, and dark curtains hang over the windows. The palanquin protects its occupants from sunlight and severe heat and cold (though not extreme or incredible heat or cold) as well as blocks most non-damaging forms of weather. The palanquin is only as sturdy as a standard wooden carriage—if it would be destroyed, it instantly reverts to a square of fabric and can't be Activated again for 1 month.</p>\n<p>You can return the palanquin to its original shape by using an Interact action to command the spectral bearers to collapse it. They disappear the moment it returns to its fabric form.</p>"
```

Comment:
```suggestion
            "value": "<p>This small square of dark blue fabric appears to be painted with a depiction of the night sky, shifting each day to match the sky above Geb the previous night.</p>\n<p><strong>Activate</strong> <span class=\"action-glyph\">3</span> Interact</p>\n<hr />\n<p><strong>Effect</strong> The fabric unfolds into a dark palanquin large enough to comfortably hold two Medium creatures. Four spectral bearers appear at the corners of the palanquin, capable of carrying it effortlessly over most types of terrain as long as the terrain is relatively flat and devoid of hazards. Spectral bearers have the statistics of an @UUID[Compendium.pf2e.pathfinder-bestiary.Actor.Unseen Servant], except they're strong enough to carry the palanquin and its occupants. They move generally along a basic set of directions you indicate upon activation, which can include turns onto various streets, but they don't take other creatures' feelings into account and can be inconsiderate to passersby in an inhabited area as they relentlessly move you forward. The bearers move slowly but steadily along their simple directions. While the palanquin is perfectly suitable for overland travel, the bearers' imprecise movements and the requirement to reactivate the palanquin to change destinations make it unsuited for a combat encounter or other situation where seconds and precise movements count.</p>\n<p>The palanquin's interior ceiling is painted with the same night sky pattern as the cloth, and dark curtains hang over the windows. The palanquin protects its occupants from sunlight and severe heat and cold (though not extreme or incredible heat or cold) as well as blocks most non-damaging forms of weather. The palanquin is only as sturdy as a standard wooden carriage—if it would be destroyed, it instantly reverts to a square of fabric and can't be Activated again for 1 month.</p>\n<p>You can return the palanquin to its original shape by using an Interact action to command the spectral bearers to collapse it. They disappear the moment it returns to its fabric form.</p>"
```